Dockerfile

```diff
@@ -57,11 +57,18 @@ RUN mkdir /src && \
     luarocks install lua-requests
 
 ADD . /src/
+
+WORKDIR /src/samples
+RUN make
+
 WORKDIR /src
 
 RUN make && \
     cp -r lib /opt/nginx/ && \
     cp nginx/conf/nginx.conf /opt/nginx/conf/nginx.conf
 
+RUN chmod 755 bin/webtorch
```

Comment:
Better to add it to the above RUN as it will add another layer to the docker image ( a fair amount of space)

Reply:
make sense, updated